app/views/help.scala.html

```diff
@@ -19,19 +20,23 @@ <h2 class="govuk-heading-m">This user help guide includes:</h2>
                     <li><a href="#registering-new-account">Registering for a new account</a></li>
                     <ul class="govuk-list govuk-list--bullet">
                         <li><a href="#new-user-email">New user email</a></li>
-                        <li><a href="#set-up-two-step-auth">Set up your OTP account for multi-factor authentication</a></li>
+                        <li><a href="#set-up-two-step-auth">Set up your OTP account for Multi-Factor Authentication</a></li>
                         <li><a href="#set-password">Set your password</a></li>
                     </ul>
                     <li><a href="#using-existing-account-holder">Using the service as an existing account holder</a></li>
                     <ul class="govuk-list govuk-list--bullet">
-                        <li><a href="#existing-web-browsers">Web browsers</a></li>
                         <li><a href="#access-tdr">Accessing TDR</a></li>
-                        <li><a href="#sign-in-mfa">Sign in to your account using multi-factor authentication</a></li>
-                        <li><a href="#select-series">Step one: Select series reference</a></li>
-                        <li><a href="#transfer-agreement">Step Two: Transfer agreement</a></li>
-                        <li><a href="#upload-records">Step three: Upload your records</a></li>
-                        <li><a href="#file-checks">Step four: File checks</a></li>
-                        <li><a href="#confirm-transfer">Step five: Confirm transfer</a></li>
+                        <li><a href="#sign-in-mfa">Sign in to your account using Multi-Factor Authentication</a></li>
+                    </ul>
+                    <li><a href="#step-by-step-guide">A step-by-step guide to transferring your record</a></li>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li><a href="#select-series">Step 1: Select series reference</a></li>
+                        <li><a href="#transfer-agreement">Step 2: Transfer agreement</a></li>
+                        <li><a href="#upload-records">Step 3: Upload your records</a></li>
+                        <li><a href="#file-checks">Step 4: File checks</a></li>
+                        <li><a href="#additional-metadata">Step 5: Add, edit or delete metadata</a></li>
+                        <li><a href="#download-metadata">Step 6: Download and review metadata</a></li>
+                        <li><a href="#confirm-transfer">Step 7: Confirm transfer</a></li>
                         <li><a href="#transfer-complete">Transfer complete</a></li>
                     </ul>
                     <li><a href="#glossary">Glossary</a></li>
```

Comment:
Should add the `View the progress of your transfers and transfer history` section to the contents table.